### PR TITLE
batch size stuff for val

### PR DESCRIPTION
**Pull request by andrewwangva**

### CODE REVIEW

File: src/training/vqa_fine_tune.py [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+
+class CLIPMultimodalClassifier(nn.Module):
+    def __init__(self, encoder, embed_dim, num_labels):
+        super().__init__()
+
+        self.encoder = encoder
+        
+        self.fc1 = nn.Linear(embed_dim * 2, 1536) #size of answer space
+        self.lnorm = nn.LayerNorm(1536)
+        self.fc2 = nn.Linear(1536, num_classes)
```
Comment (apsdehal) [inline src/training/vqa_fine_tune.py]: Taking `num_labels` in but assigning `num_classes`?

File: src/training/vqa_fine_tune.py [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+    def __getitem__(self, idx):
+        item = self.df.iloc[idx]
+        img_path = item["image"]["path"]
+        image = Image.open(str(img_path))
+        text = item["question"]
+        label = self.labels[idx]
+        return {
+            'image': self.transforms(image),
+            'text': self.tokenize([text])[0],
```
Comment (apsdehal) [inline src/training/vqa_fine_tune.py:42]: You would need the attention mask from this no? Or is it automatically calculated on the fly?

File: src/training/vqa_fine_tune.py (hunk 1 above)
Comment (andrewwangva) [reply src/training/vqa_fine_tune.py]: whoops! That's a weird typo hmm...

File: src/training/vqa_fine_tune.py [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+        
+        self.fc1 = nn.Linear(embed_dim * 2, 1536) #size of answer space
+        self.lnorm = nn.LayerNorm(1536)
+        self.fc2 = nn.Linear(1536, num_classes)
+    def forward(self, image, text):
+        # CLIP doesn't have a multimodal encoder, so we concatenate the features
+        text_features = self.encoder.encode_text(text)
+        image_features = self.encoder.encode_image(image)
+        multimodal_features = torch.cat([image_features, text_features], dim=-1)
+        layer = self.lnorm(F.relu(self.fc1(multimodal_features)))
```
Comment (gmittal) [inline src/training/vqa_fine_tune.py]: can we make this an `nn.Sequential`?
